Add Row Details documentation. (#604)
telerik/maui-docs: controls/datagrid/row-details/row-details-template.md

---
title: Row Details Template
page_title: .NET MAUI DataGrid Documentation - Row Details Template

> description: Learn how to present additional information about DataGrid row by using row details functionality.

Reviewer: That's a good description!
```suggestion
description: Learn how to present additional information about a DataGrid row by using the row details functionality.
```

position: 
slug: datagrid-row-details-template
---

# .NET MAUI DataGrid Row Details Template

> The Telerik UI for .NET MAUI DataGrid control define a `RowDetailsTemplate` property (type `DataTemplate`). The property is used for displaying row details. 

Reviewer: ```suggestion
The Telerik UI for .NET MAUI DataGrid control defines a `RowDetailsTemplate` property (type `DataTemplate`). The property is used for displaying row details. 
```

> ## Example 

Reviewer: To be added later? 
You don't need a dedicated section for the example. Adding a sentence that introduces the example is the better approach in such short articles.

---
title: Row Details Template
page_title: .NET MAUI DataGrid Documentation - Row Details Template
description: Learn how to present additional information about a DataGrid row by using the row details functionality.
position: 
slug: datagrid-row-details-template
---

# .NET MAUI DataGrid Row Details Template

The Telerik UI for .NET MAUI DataGrid control defines a `RowDetailsTemplate` property (type `DataTemplate`). The property is used for displaying row details. 

> ## Example 

Reviewer: ```suggestion
```

> This example provides steps how to define the `RowDetailsTemplate` property in the DataGrid control.

Reviewer: ```suggestion
The following example shows how to define the `RowDetailsTemplate` property in the DataGrid control.
```

1. Define the DataGrid in XAML:

<snippet id ='datagrid-rowdetails'/>

2. Define `RowDetailsTemplate` in XAML

<snippet id ='datagrid-rowdetails-template'/>

3. Define the Business Model:

<snippet id ='datagrid-business-model'/>

4. Define the ItemsSource:

<snippet id ='datagrid-items-source'/>

> 5. In addition you can define the row details to be expanded:

Reviewer: ```suggestion
5. In addition, you can define the row details to be expanded:
```
